site/content/3.10/data-science/adapters/arangordf-adapter/_index.md

```diff
@@ -33,6 +33,7 @@ Watch this
 [lunch & learn session](https://www.arangodb.com/resources/lunch-sessions/graph-beyond-lunch-break-2-11-arangordf/) to get an
 introduction on ArangoRDF - an RDF adapter developed with the community
 as a first step at bringing RDF graphs into ArangoDB.
+- Note: Since the recording of this session, ArangoRDF has been overhauled and extended. Package usage has changed, so please refer to the [Quickstart](#quickstart) section below for the latest specification.
```

Comment:
Should we remove the video link altogether?
Does the Jupyter notebook require updates too?

Reply:
> Should we remove the video link altogether?
Confirmed with @cw00dw0rd that it would be best to remove the Lunch & Learn for now until a new video is recorded (when I get the chance). How should we proceed with this? I can remove this text blurb from the markdown files, but I am not sure how we can remove https://www.arangodb.com/resources/lunch-sessions/graph-beyond-lunch-break-2-11-arangordf/

> Does the Jupyter notebook require updates too?
That won't be necessary as we've already updated the notebook in the ArangoRDF repository, so all good

Reply:
I think we can merge this PR. Regarding the lunch session itself, please let the marketing team know.

Reply:
understood, feel free to approve & merge